pkg/db/tiedot/task.go

```diff
@@ -175,6 +175,10 @@ func (d *Database) AllTasks(config *setting.Config) []agenttasks.Task {
 	return tasks_id
 }
 
+func (d *Database) AllTasksFiltered(config *setting.Config, f dbcommon.TaskFilter) (dbcommon.TaskResult, error) {
+	return dbcommon.TaskResult{}, errors.New("not implemented")
```

Comment:
Can you fix this, please? Also with a solution slow but atm I prefer to maintain both. It's easier to use Tiedot in the GitHub pipeline for testing.